src/assets/data/workshops-info.js

```diff
@@ -57,15 +57,15 @@ let workshopsData = [
         description: "In the current job market, landing your first job might be one of the most challenging experiences you may encounter to date. In this session, gain tips and tricks to help you improve your chances. You'll learn how to search for the right company/manager, find the right role, prepare for the interview and more.",
         host: {
             name: "Cox",
-            link: "https://jobs.coxenterprises.com/businesses/cox-communications/"
+            link: "https://www.coxautoinc.com/"
         },
     },
     {
         title: "Info Session + Early Career Opportunities", 
         description: "Come hear about Slalom! An overview of the company, early career opportunities for students, and a day-in-the-life of a Slalom software engineer will be provided!",
         host: {
             name: "Shalom",
```

Comment:
This should be Slalom too

Reply:
Lmao I thought the "l" in "Slalom" was an "h" the entire time. my bad